prime-router/metadata/valuesets/covid-19.valuesets

```diff
@@ -189,6 +189,19 @@
       display: Nasal structure (body structure)
       version: 2020-09-01
 
+- name: covid-19/yesno
```

Comment:
This YES/NO/UNK are the standard for the HHS data, not hl70136.   I didn't see a Valueset that was quite exactly that, so I ended up creating this one.